web/core/src/jsMain/kotlin/org/jetbrains/compose/web/attributes/AttrsBuilder.kt

```diff
@@ -43,6 +48,14 @@ open class AttrsBuilder<TElement : Element> : EventsListenerBuilder() {
         styleBuilder.apply(builder)
     }
 
+    fun style(propertyHolder: Any) {
+        styleBuilder.apply {
+            Object.entries(propertyHolder).forEach { (k, v) ->
```

Comment:
Did you want to try with `Object.assign()`? 
I seems that with `Object.assign()` we can avoid having `forEach` loop.